src/Sylius/Bundle/CoreBundle/spec/Validator/Constraints/ImageUniqueCodeValidatorSpec.php

```diff
@@ -1,68 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace spec\Sylius\Bundle\CoreBundle\Validator\Constraint;
```

Comment:
This was not even run by PhpSpec because the namespace is missing "s" at the end.